apps/mobile/src/app/(learner)/session/index.test.tsx

```
@@ -287,6 +322,104 @@ describe('SessionScreen homework flow', () => {
     );
   });
 
+  // TODO: agent was stopped mid-implementation — unskip when chips are wired
+  it.skip('shows contextual learner-agency chips and session tools', () => {
+    render(<SessionScreen />);
+
+    expect(screen.getByText('I know this')).toBeTruthy();
```

ESLint

[error] Unexpected use of 'screen'.

(no-restricted-globals)